client/src/home/a11y-home.yaml

```diff
@@ -29,14 +29,14 @@ home_a11y_subject_section_title:
 home_a11y_non_a11y_redirect_warning:
   transform: [markdown]
   en: |
-    You have been redirect to the text based InfoBase from a page with no text-only equivalent. 
+    You have been redirect to the text-only InfoBase from a page with no text-only equivalent. 
     Pages with no text-only equivalent are purely for visualization purposes and are not the sole presentation of any data in the InfoBase.
     Any information which was presented on the page you were redirected from can be found in the [report builder](#rpb).
   fr: |
-    Vous avez été redirigé vers une version d'InfoBase textuel à partir d'une page sans équivalent textuel.
+    Vous avez été redirigé vers une version d'InfoBase texte seulement à partir d'une page sans équivalent texte seulement.
```

Comment:
I don't think this change was necessary and "version d'InfoBase texte seulement" doesn't make sense